twitter/tests/test.bal

```diff
@@ -126,6 +126,18 @@ function testGetUserTimeline() returns error? {
 @test:Config {
     dependsOn: [testGetUserTimeline], enable: true
 }
+function testGetUserTweets() returns error? {
+    log:printInfo("testGetUserTweets");
+    Tweet[] tweetResponse = check twitterClient->getUserTweets(screenName);
+    test:assertTrue(true, "Failed to call getUserTweets()");
```

Comment:
This line is not useful.
If we want to assert, we need to add some condition as the 1st parameter. But we are adding `true`. So always its gonna be true.

Reply:
We can remove that line right?

Reply:
Ack. Updated